IfcDiscreteAccessory Rail Import BIMQ 2021-07-21T07:41:33.5120880Z (#187)
bSI-InfraRoom/IFC-Specification: IFC4x3/Sections/Domain specific data schemas/Schemas/IfcRailDomain/PropertySets/Pset_DiscreteAccessoryTypeRailBrace/Documentation.md

> Properties of rail brace used in railway. The property set can be used by the predefined type RAILBRACE of IfcDiscreteAccessory

Reviewer: ```suggestion
Properties of rail brace used in railway. The property set can be used by the predefined type RAILBRACE of IfcDiscreteAccessory.
```

> Properties of rail brace used in railway. The property set can be used by the predefined type RAILBRACE of IfcDiscreteAccessory.

Reviewer 2: ```suggestion
Properties of a rail brace. The property set can be used by the predefined type RAILBRACE of IfcDiscreteAccessory.
```
